src/components/App/App.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
+
 import ThemeProvider from '@material-ui/styles/ThemeProvider';
 import CssBaseline from '@material-ui/core/CssBaseline';
-import Typography from '@material-ui/core/Typography';
 
+import Root from '../Root';
 import theme from '../../styles/theme';
 
 const App: React.FC = () => {
   return (
     <ThemeProvider theme={theme}>
       <CssBaseline />
-      <Typography>Buddy App</Typography>
+      <Root />
```

Comment:
TBH I don't like this name as it's not clear _what it does_. I guess `<Container />` would be the best, but it's probably namespace collision with `import Container from '@material-ui/core/Container';` Maybe `AppContainer`  / or `Layout`?

Reply:
I will call it `AppWrapper` since container and wrapper are similar names :) I didn't want to call it `AppContainer` this way due the collision with Redux naming that is also using `Container.